packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -613,6 +629,14 @@ describe('AddresslistVoting', function () {
         expect(proposal.tally.abstain).to.equal(1);
       });
 
+      it('reverts on voting None', async () => {
+        await advanceIntoVoteTime(startDate, endDate);
+
+        await expect(voting.vote(id, VoteOption.None, false))
```

Comment:
The same here as above.

Reply:
Added in 0fed43072dcd874cd9b4d43a70e0a28bc95a39e0